tests/t_json

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
All three test cases looks similar, can you move into sh/bash function/functions?
To simplify ading new test cases in the future?